instance/templates/instance/ansible/vars.yml

```diff
@@ -181,11 +181,33 @@ POSTFIX_QUEUE_SENDER_CANONICAL_MAPS: '{{ smtp_relay_settings.source_address }}
 {% endif %}
 {% endwith %}
 
+
+
+# EDXAPP_CMS_NGINX_PORT: 80
```

Comment:
I think this should be uncommented rather than removed.  Why did you comment it out?